engine/runtime-compiler/src/main/java/org/enso/compiler/context/CompilerContext.java

```diff
@@ -86,6 +88,8 @@ void initializeBuiltinsIr(
 
   CharSequence getCharacters(Module module) throws IOException;
 
+  IdMap getIdMap(Module module);
```

Comment:
Why do you need this method, when there also is:
```java
public abstract IdMap getIdMap();
```
on the `Module` itself? Looks like unnecessary duplication.